docs/docs/scroll/useScrollViewOffset.mdx

```diff
@@ -59,12 +59,16 @@ import ScrollViewOffsetSrc from '!!raw-loader!@site/src/examples/ScrollViewOffse
   showCode
 />
 
+## Remarks
+
+- You can pass the `animatedRef` argument conditionally and hook will correctly return values basing on the ScrollView component it is connected to: `useScrollViewOffset(someState ? someScrollRefA : someScrollRefB)`.
```

Comment:
This is true only in Web implementation - I don't think it will work properly on native platforms. Let's remove this annotation for now and add it once the whole hook can perform it.

Reply:
@kacperkapusciak @tjzel seems like it works on native (both iOS and Android, attaching iOS example) 😄 

https://github.com/software-mansion/react-native-reanimated/assets/77503811/dfc99dc1-7f82-4427-8fd6-f2bce658a128

Reply:
Let's rewrite this sentence. Instead of talking of "conditional" argument let's specify, that the reference passed to the hook can be changed at will (example being conditional call).

Reply:
@tjzel on it 👍